### ISSUE TITLE
Adjusting Subtitles 

### ISSUE THREAD

**Issue by vinhdu0212** [gignupg/Movie-Subtitles#14]
I currently watching a show on Netflix and was wondering how to change the positioning of the subtitles so that it's the same as Netflix's subtitles.

**Comment by gignupg** [comment]
Can't you drag and drop the subtitles into your prefered position?

**Comment by vinhdu0212** [comment]
No, I can't. My subtitles on Netflix are stuck at the very bottom

**Comment by gignupg** [comment]
Hmm, thanks for reporting this! I will try to fix this if I have more time at some point in the future. Until then, if someone wants to make a pull request I'll gladly accept it!
